### ISSUE TITLE
keep getting this types of command for rarest poke?

### ISSUE THREAD

**Issue by akuengkau** [NoxxDev/NecroBot#688]
![capture](https://cloud.githubusercontent.com/assets/21312667/18054948/9ad03044-6e38-11e6-812d-036b0da1174d.JPG)

**Comment by iwean** [comment]
Nope, sometime feeder site have false/fake location for the rare...

hmm.... @samuraitruong how about just retrieve verified only?

**Comment by hardknoxni69a** [comment]
Yes probably not verified unfortunately theres a few feeds that have alot of fakes atleast u just walked out of your way a little rather then sniping across country for nothing

**Comment by kalamera** [comment]
I have one account sniping all the way to lvl 20 without any softban.
Not sure how longer it will last, but the long distance sniping does not seem to be an issue so far.
If have filtered the special pokemons, but it happens there are some hiding on fake locations.

**Comment by samuraitruong** [comment]
all my long distance snipe account got ban when reach 30. that why i make human walk snipe. I can collect 1k draniti candy in 1 week. I don't  need long distance snipe any more.
